docs/src/components/pricing/PricingTable.tsx

```diff
@@ -636,6 +663,23 @@ const rowHeaders: Record<string, React.ReactNode> = {
 const yes = <IconImage name="pricing/yes" title="Included" />;
 const pending = <IconImage name="pricing/time" title="Work in progress" />;
 const no = <IconImage name="pricing/no" title="Not included" />;
+const toBeDefined = (
+  <Typography
+    component={Link}
+    href="https://forms.gle/19vN87eBvmXPjBVp6"
```

Comment:
It seems that I can't access this form https://forms.gle/19vN87eBvmXPjBVp6. 

Can we share access to this Google Form with all@mui.com with write access? (or at minimum view and product@mui.com with write access) Thanks.

Reply:
Thanks